BioCro/DARPA/biocro_1.0_darpa.Rmd

```diff
@@ -0,0 +1,326 @@
+---
+title: "Using BioCro 1.0 on Setaria Data"
+author: "Author: Kristina Riemer"
+output: github_document
+urlcolor: blue
+---
+
+### Switching between BioCro versions locally
+
+Needs to be run using BioCro version 1.0. Check your BioCro version: 
+
+```{r, message=FALSE}
+library(BioCro)
+if(!packageVersion(pkg = 'BioCro') >= 1.0){
+  warning("need to use BioCro v1.0 or greater")
+} else {
+  #devtools::install_github('ebimodeling/biocro-dev')
+  devtools::install_github('ebimodeling/biocro@new-framework')  
+}
+
+```
+
+### Read in data
```

Comment:
Needs to be more clear where the data are from - this assumes I have run the biocro_biomass_darpa.Rmd script, and for that script that I have manually downloaded a csv file and put it in the root directory of the repository. 

we should be able to make use of a private place to put the data (either using googlesheets to access the current spreadsheet or using a private github repository - I prefer the private repository idea)

Reply:
This is definitely not the most clear or reproducible due to data restrictions, so I appreciate your suggestions. I intended to use googlesheets, but UA doesn't allow me to authenticate my Google Drive. I agree with using a private repo to store the data, but are we (me?) going to have to make sure the repo is up to date with the Google Drive data?

Reply:
yes - put in a private repo and make sure data are up to date w/ google drive. You have to download a new csv every time you make changes anyhow, so this shouldn't be that much more difficult (and would have the added benefit of version control and having everything in one place!).

Reply:
I created [an new issue](https://github.com/danforthcenter/sentinel-detection/issues/141) for this task because it's going to apply across other vignettes in addition to this one.